inc/config.class.php

```diff
@@ -1917,6 +1917,15 @@ function showSystemInformations() {
 
       echo "\n</pre></td></tr>";
 
+      echo "<tr><th>GLPI constants</th></tr>\n";
+      echo "<tr class='tab_bg_1'><td><pre>\n&nbsp;\n";
+      foreach (get_defined_constants() as $constant_name => $constant_value) {
+         if (preg_match('/^GLPI_/', $constant_name)) {
+            echo $constant_name . ': ' . $constant_value . "\n";
+         }
+      }
+      echo "\n</pre></td></tr>";
+
```

Comment:
I used this to make testing easier, but should we keep this "feature" ?

Reply:
Sure, it may be of interest :)